src/types.hpp

```diff
@@ -7,9 +7,12 @@
 
 // Type declarations for node ids and way ids.  We don't need the full 64 bits
 // for ways as the highest way number isn't close to 2^32 yet.
-typedef std::uint64_t external_nodeid_t;
+typedef std::int64_t external_nodeid_t;
```

Comment:
Because NaN doesn't like uint64_t:
```
/.deps/Release/obj.target/route_annotator/src/lookup_bindings.o.d.raw   -c -o Release/obj.target/route_annotator/src/lookup_bindings.o ../src/lookup_bindings.cpp
../src/lookup_bindings.cpp:68:19: error: no matching function for call to 'To'
    auto nodeId = Nan::To<external_nodeid_t>(jsNodeId).FromJust();
                  ^~~~~~~~~~~~~~~~~~~~~~~~~~
/Users/kajari/Mapbox/route-annotator/node_modules/nan/nan_converters.h:53:38: note: candidate template ignored: substitution failure [with T = unsigned long long]: implicit instantiation of undefined template 'Nan::imp::ToFactory<unsigned long long>'
typename imp::ToFactory<T>::return_t To(v8::Local<v8::Value> val) {
~~~~~~~~
```